mk/crypto.mk

```diff
@@ -1,28 +1,20 @@
 # SPDX-License-Identifier: Apache-2.0
 LDLIBS += -L$(LIB_DIR)
 
-LIBDEPS += $(LIB_DIR)/libfips202.a
+LIBDEPS += $(LIB_DIR)/libfips202.a $(LIB_DIR)/libnistrng.a $(LIB_DIR)/librng.a
```

Comment:
Remind me, why are we exporting a librng in the first place? We only offer an insecure test-only randombytes, and I don't think we should be exporting this? Otherwise, we risk people linking against this in production builds. Or am I missing something @mkannwischer @potsrevennil

Reply:
Discuss this elsewhere